package.json

```diff
@@ -23,9 +25,11 @@
     "node": ">=6.9.5"
   },
   "dependencies": {
+    "@types/node": "^9.4.6",
     "check-type": "^0.4.11",
     "js-yaml": "^3.10.0",
-    "optionator": "^0.8.2"
+    "optionator": "^0.8.2",
+    "typescript": "^2.7.2"
```

Comment:
Are these needed always? Perhaps just in the development environment?

Reply:
I probably used the wrong command. They should be devDependencies. I'll fix it.